phpstan-baseline.neon

```diff
@@ -0,0 +1,36 @@
+parameters:
```

Comment:
Why these errors are ignored?

Reply:
Because phpstan 1.10 is a major pain on level 9 when dealing with array properties that might contain 6 different factory interface types, and each factory method should return a different type but they all rely on the same getFactory() method behind it.

If you want to do static analysis 100% correctly that would require someone more familiar with phpstan and with time to spend fighting and nit-picking with the analysis tool - which hardly seems worth it at this point...